aliases/available/vagrant.aliases.bash

```diff
@@ -2,13 +2,18 @@ cite 'about-alias'
 about-alias 'vagrant aliases'
 
 # Aliases
+alias vhl='vagrant hosts list'
+alias vscp='vaagrant scp'
+alias vsl='vagrant snapshot list'
+alias vst='vagrant snapshot take'
 alias vup="vagrant up"
 alias vupl="vagrant up 2>&1 | tee vagrant.log"
 alias vh="vagrant halt"
 alias vs="vagrant suspend"
 alias vr="vagrant resume"
 alias vrl="vagrant reload"
 alias vssh="vagrant ssh"
+alias vsshr="vagrant ssh $1 -- -t 'sudo su -; /bin/bash'"
```

Comment:
Same here - aliases don't take parameters. This is not going to work. Might be better suited for a function.